validation/src/test/scala/hmda/validation/rules/lar/validity/V455Spec.scala

```diff
@@ -0,0 +1,39 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.validation.dsl.{ Failure, Success }
+import hmda.validation.rules.lar.LarEditCheckSpec
+import org.scalacheck.Gen
+
+class V455Spec extends LarEditCheckSpec {
+  import Gen.posNum
+
+  property("Succeeds when Applicant Ethnicity is not 1, 2, or 3") {
+    forAll(larGen, posNum[Int]) { (lar, ethn) =>
+      whenever(ethn > 3) {
+        val applicant = lar.applicant.copy(ethnicity = ethn)
+        val newLar = lar.copy(applicant = applicant)
+        V455(newLar) mustBe a[Success]
+      }
+    }
+  }
+  property("Succeeds when Applicant Race1 is not 7") {
+    forAll(larGen, posNum[Int]) { (lar, race) =>
```

Comment:
As above, I agree those are the only _valid_ values, but they're not the only possible values. I want the test to reflect more cases than just the happy path.